docs/architecture/baking.md

```diff
@@ -27,25 +27,62 @@ It packages pending operations into a block, signs it, and distributes it to the
 
 ## Becoming a baker
 
-To become a baker, you must create an account to act as a _delegate_, which is an account that is authorized to bake blocks and attest blocks that other accounts bake.
+To become a baker, you must create an account to act as a _delegate_, which is an account that is authorized to bake blocks and attest blocks that other accounts bake, and also to receive delegations from other accounts (as explained later).
```

Comment:
delegations and stake from other accounts.

We should move the introduction of the term _delegate_ up front, when we describe the baker role.

Reply:
I added "delegations (and stake)".

I don't think we should move the definition earlier, because above we only explain the roles of a baker: creating and (pre)attesting blocks. The role as a delegate comes into play only here.

The whole idea of this page is to present a more accessible view of Tezos, omitting full details from the start. I think Tim did a great job when writing this page, and we should not overload it too much. But of course, we have to avoid stating things that are (too) false.